sdk/examples/client/block/04_block_tagged_data.rs

```diff
@@ -46,9 +46,9 @@ async fn main() -> Result<()> {
         )
         .await?;
 
-    println!("{block:#?}\n");
+    println!("{wrapper:#?}\n");
 
-    if let Some(Payload::TaggedData(payload)) = block.payload() {
+    if let Some(Payload::TaggedData(payload)) = wrapper.block().as_basic().payload() {
```

Comment:
Should we maybe put the `as_basic` etc on the wrapper as well, just for ease of access?

Reply:
Done